backend/googlecloudstorage/googlecloudstorage.go

```diff
@@ -51,9 +52,10 @@ const (
 	rcloneClientID              = "202264815644.apps.googleusercontent.com"
 	rcloneEncryptedClientSecret = "Uj7C9jGfb9gmeaV70Lh058cNkWvepr-Es9sBm0zdgil7JaOWF1VySw"
 	timeFormatIn                = time.RFC3339
-	timeFormatOut               = "2006-01-02T15:04:05.000000000Z07:00"
-	metaMtime                   = "mtime" // key to store mtime under in metadata
-	listChunks                  = 1000    // chunk size to read directory listings
+	timeFormatOut               = time.RFC3339Nano
```

Comment:
`RFC3339Nano` removes trailing 0s which aren't used whereas the current format doesn't. Why do you wish to make this change?

Reply:
I didn't realize the current format doesn't remove trailing zeros. But removing them shouldn't cause any problems. If you're concerned, I can revert this change.

Reply:
I don't think it will cause problems either - lets leave your change alone.